tests/e2e/helpers/decompressors/brotli-database-populator.ts

```diff
@@ -0,0 +1,122 @@
+import * as proto from 'protobufjs';
+import { pack as msgpackrPack } from 'msgpackr';
+import * as brotli from 'brotli-unicode';
+import * as fflate from 'fflate';
+import * as fs from 'fs';
+import { BaseDatabasePopulator } from './base-decompressors-populator';
+
+const COMPRESSED_PREFIX = 'Comp';
+const BROTLI_PREFIX = 'BROTLI';
+
+export class BrotliDatabasePopulator extends BaseDatabasePopulator {
+
+    protected async createCompressedKeys(): Promise<void> {
+        await this.createBrotliUnicodeKeys();
+        await this.createBrotliASCIIKeys();
+        await this.createBrotliVectorKeys();
+        await this.createBrotliJSONKeys();
+        await this.createBrotliPHPUnserializedJSONKeys();
+        await this.createBrotliJavaSerializedObjectKeys();
+        await this.createBrotliMsgpackKeys();
+        await this.createBrotliProtobufKeys();
+        await this.createBrotliPickleKeys();
+    }
+
+    private async createBrotliUnicodeKeys() {
+        const encoder = new TextEncoder();
+        const prefix = `${COMPRESSED_PREFIX}:${BROTLI_PREFIX}:Unicode`;
+        const rawValue = '漢字';
+        const buf = encoder.encode(rawValue);
+        const value = Buffer.from(await brotli.compress(buf));
+        await this.createString(prefix, value);
```

Comment:
you are adding here only a string key
you have also createHash for now

I understand that you are going to add all other key types here

Reply:
fixed